src/shared/components/graph_tips/GraphTips.tsx

```diff
@@ -20,21 +19,14 @@ const GraphTips: FC = () => (
           <div className="graph-tips-line">
             <code>Double Click</code> <span>Reset Graph Window</span>
           </div>
-          <FeatureFlag name="annotations">
-            <br />
-            <span>When Annotations button is on:</span>
-            <div className="graph-tips-line">
-              <code>Shift + Click</code>{' '}
-              <span> Annotate a point on a graph</span>
-            </div>
-          </FeatureFlag>
-
-          <FeatureFlag name="rangeAnnotations">
-            <div className="graph-tips-line">
-              <code>Shift + Drag</code>{' '}
-              <span> Annotate a range on a graph </span>
-            </div>
-          </FeatureFlag>
+          <br />
+          <span>When Annotations button is on:</span>
+          <div className="graph-tips-line">
+            <code>Shift + Click</code> <span> Annotate a point on a graph</span>
```

Comment:
why did you remove the spacing? {' '}
(both here and below).  it still looks good/lines up?

Reply:
@jrenee42 prettier did that, not me